extensions/components/com_redcore/site/language/da-DK/da-DK.com_redcore.ini

```diff
@@ -0,0 +1,24 @@
+; redCORE
```

Comment:
Since I'm not entirely sure which language files you are referring to I'll adress both:

The general joomla admin language files are needed, since the parameters translation type will be using them.

The site com_redcore language files are, now that I think about it, actually not needed. I just need to load the com_redcore language files. This is probably a good idea, to avoid having duplicate language specifications, so I'll do that instead.